src/domainModules/collectionMammals/components/MammalForm/index.spec.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+import React from 'react'
+import setupTestComponent from 'utilities/test/setupTestComponent'
+import MammalForm from 'domainModules/collectionMammals/components/MammalForm'
+
+describe('domainModules/collectionMammals/components/MammalForm', () => {
+  it('renders without crashing', () => {
+    setupTestComponent({
+      component: <MammalForm handleFormSubmit={() => {}} />,
+      mount: true,
+    })
+  })
+
+  it('Is initialized in form state', () => {
+    const { store } = setupTestComponent({
+      component: <MammalForm handleFormSubmit={() => {}} />,
+      fullExport: true,
+      mount: true,
+    })
+
+    expect(store.getState().form.mammalForm).toBeTruthy()
+  })
+
+  it('Submit fail when cataloged number not provided', () => {
```

Comment:
cataloged number => catalog number (also in test below, because it can be confusing what it's called with catalogedUnit and catalogNumber)